README.md

```diff
@@ -74,7 +65,7 @@ The compiled SDK size is ~155kb. You can clone this repository to keep up with t
 
 Branch is available through [CocoaPods](http://cocoapods.org). To install it, simply add the following line to your Podfile:
 
-```objc
```

Comment:
@jakecadams has requested we keep the language in code tags for the docs portal.